crates/libcgroups/src/v2/devices/mod.rs

```diff
@@ -3,4 +3,9 @@ pub mod controller;
 pub mod emulator;
 pub mod program;
 
+#[cfg(test)]
+#[allow(clippy::too_many_arguments)]
+#[cfg_attr(coverage, feature(no_coverage))]
+pub mod mocks;
```

Comment:
only used in tests, also libc functions trigger the `too_many_arguments` clippy warning